jar-filter/build.gradle

```diff
@@ -22,6 +22,21 @@ gradlePlugin {
     automatedPublishing = false
 }
 
+pluginBundle {
+    website = project.vcsUrl
+    vcsUrl = project.vcsUrl
+    description = 'Corda JAR Filter Plugin'
+    tags = ['corda']
+
+    plugins {
+        jarFilterPlugin {
+            displayName = 'Corda JAR Filter Plugin'
+        }
+    }
+}
+
+task('publishPlugin').dependsOn jar
```

Comment:
This is incorrect. It should actually depend on the `shadowJar` task defined below.